CONTRIBUTING.rst

```diff
@@ -336,13 +336,19 @@ To run the tests, use the following command:
 How to add new dependency
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Install the requirements package.
-
-The flag -d will install the development requirements.
+Install the new package as a dependency.
 
+If you are adding a new package that is only needed for development, use the
+``-d`` flag to install it as a development dependency.
 .. code:: shell
```

Comment:
We need add a empty line here.
See the rendered version here https://repository-service-tuf--882.org.readthedocs.build/en/882/devel/contributing.html?readthedocs-diff=true&readthedocs-diff-chunk=1#how-to-add-new-dependency